odbc/src/handles/definitions.rs

```diff
@@ -200,6 +205,7 @@ impl Connection {
 pub struct Statement {
     pub connection: *mut MongoHandle,
     pub mongo_statement: Option<Box<dyn mongo_odbc_core::MongoStatement>>,
+    pub var_data_indicies: Option<HashMap<USmallInt, (usize, usize)>>,
```

Comment:
This maps the column to the current index position in the column with a total size (that is only non-zero for WChar data). Having to store the total size is annoying, but I can't find a more efficient way to handle this.

Reply:
yes! Though, now I'm thinking it's better to store the data in there so that we aren't calling the `get_data` api multiple times since it copies the data out of the current row every time.